packages/components/src/components/index.ts

```diff
@@ -1 +1,2 @@
+export * from './layout';
```

Comment:
Should we rename layout to layouts to be consistent with plural?

Reply:
done!